packit_service/worker/jobs.py

```diff
@@ -615,10 +618,19 @@ def get_jobs_matching_event(self) -> List[JobConfig]:
                 )
                 and job not in jobs_matching_trigger
             ):
-                jobs_matching_trigger.append(job)
+                if (
+                    not job.manual_trigger
+                    or any(
+                        isinstance(self.event, event_type)
+                        for event_type in MANUAL_EVENTS
+                    )
+                    or job.trigger != JobConfigTriggerType.pull_request
```

Comment:
Done :)

Reply:
is the `job.trigger != JobConfigTriggerType.pull_request` acutally needed?

Reply:
in case user will specify `manual_trigger=True` in jobs that do not care about the option we should skip it...now I am thinking if checking job type is a better option or not 🤔  something like `job.type != JobType.tests`...or is this all handled by decorators here https://github.com/packit/packit-service/blob/38d6158d1b718a079d4b08c35bbd69caf2e7859b/packit_service/worker/handlers/testing_farm.py#L70 ?

Reply:
yes, the decorators should take care of the mapping itself

my thinking was that by omitting that part of the condition this could be used generally for any job (e.g. also for the mentioned propose-downstream) without binding it to the testing farm job (+ the docs would explain the option properly); @mfocko WDYT?

Reply:
+1 for having it for all commands; if it gets used in a job that is not easily run via comment, it's a user error IMO, not an issue :shrug:

Also I looked around, not only at the diff, I would probably merge it with the `if` in which it's wrapped now; keeping the last check of the `if` above at the end even after merging it in one condition. (It just seems like an unnecessary indentation)

Reply:
Got it, I updated the code and tests.